<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diarios_polizas</class>
 <widget class="QWidget" name="diarios_polizas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QFrame" name="central_frame">
     <property name="styleSheet">
      <string notr="true">QWidget#central_frame{border: 1px solid rgb(249,66,58);}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="background_frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(151, 153, 155);

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="contenido_frame">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(179, 179, 179)</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QFrame" name="botones_frame">
            <property name="geometry">
             <rect>
              <x>520</x>
              <y>10</y>
              <width>81</width>
              <height>31</height>
             </rect>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QToolButton" name="minimizar">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>0</y>
               <width>21</width>
               <height>20</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-with 0px;</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../iconos/icons/minimize-window.png</normaloff>../iconos/icons/minimize-window.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
            </widget>
            <widget class="QToolButton" name="maximizar">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>0</y>
               <width>25</width>
               <height>19</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-with 0px;</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../iconos/icons/close-window.png</normaloff>../iconos/icons/close-window.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </widget>
           <widget class="QFrame" name="tabla_frame">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>591</width>
              <height>381</height>
             </rect>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QFrame" name="frame_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>591</width>
               <height>41</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QPushButton" name="boton_buscar">
              <property name="geometry">
               <rect>
                <x>510</x>
                <y>10</y>
                <width>71</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(116, 170, 80);</string>
              </property>
              <property name="text">
               <string>Buscar</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="numero_fin">
              <property name="geometry">
               <rect>
                <x>320</x>
                <y>10</y>
                <width>71</width>
                <height>22</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit#numero_fin{border: 1px solid rgb(0, 0, 0);}</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_8">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>10</y>
                <width>101</width>
                <height>25</height>
               </rect>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Número final:</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>12</x>
                <y>12</y>
                <width>82</width>
                <height>25</height>
               </rect>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Núm inicial:</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="numero_ini">
              <property name="geometry">
               <rect>
                <x>100</x>
                <y>10</y>
                <width>91</width>
                <height>22</height>
               </rect>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit#numero_ini{border: 1px solid rgb(0, 0, 0);}</string>
              </property>
             </widget>
            </widget>
            <widget class="QPushButton" name="boton_imprimir">
             <property name="geometry">
              <rect>
               <x>510</x>
               <y>330</y>
               <width>75</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(116, 170, 80);</string>
             </property>
             <property name="text">
              <string>Imprimir</string>
             </property>
            </widget>
            <widget class="QTableWidget" name="tabla_datos">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>50</y>
               <width>571</width>
               <height>271</height>
              </rect>
             </property>
             <property name="rowCount">
              <number>4</number>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row>
              <property name="text">
               <string/>
              </property>
             </row>
             <row/>
             <row/>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
